pyproject.toml

```
@@ -26,7 +26,7 @@ maintainers = [
      {name = "Jonatan Samoocha"},
      {name = "Yihong"},
     ]
```

```suggestion
     {name = "Yihong"},
     {name = "Émile Nadeau"},
    ]
```

@enadeau adding you here as a maintainer.